v1/content.yaml

```diff
@@ -366,9 +366,18 @@ paths:
         - application/json
       responses:
         '200':
-          description: The existing revision of the page matches the sent text
+          description: The content was not changed, and no new version was created.
+          schema:
+            type: string
```

Comment:
This is not 100% precise: in case sections property is provided, this returns JSON.